todo-frontend-angular/src/app/todo/todo.service.ts

```diff
@@ -1,17 +1,19 @@
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import { Observable, throwError } from 'rxjs';
 import { catchError, map, retry } from 'rxjs/operators';
 import { Todo } from '../models/todo.model';
 import { Todolist } from '../models/todolist.model';
+import { NewTodoDTO } from '../models/new-todo-DTO.model';
 
 
 @Injectable({
   providedIn: 'root'
 })
 export class TodoService {
   private readonly url= 'https://localhost:7058/TodoList'
+  private readonly todoUrl='https://localhost:7058/Todo'
```

Comment:
New lesson!

1. Since you now have to urls which refers ti localhost, you could use an environment variable instead! ASk for help if needed!

Reply:
I will look into it later on 🥇